pcf/core/quasiparticle.py

```diff
@@ -164,9 +164,13 @@ def set_desired_state(self, desired_state):
 
         Args:
 
-            desired_state (State):
+            desired_state (str): one of running,stopped,terminated. Can also pass in the state directly. ex. State.running
```

Comment:
This is a matter of preference, but I would personally not mention that you can use the State object directly. It would be nice to have users working with the string approach, and it opens the door to deprecating the use of the state object directly in the future when we do a major version iteration.